textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriter.java

```diff
@@ -0,0 +1,69 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import edu.uci.ics.textdb.api.common.*;
+import edu.uci.ics.textdb.api.dataflow.IOperator;
+import edu.uci.ics.textdb.common.field.DataTuple;
+import edu.uci.ics.textdb.common.field.StringListField;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by kishorenarendran on 25/04/16.
+ * Class to convert a query string into a list of possible queries using FuzzyTokenizer
```

Comment:
Add 1-2 examples to the comments to explain this operator.  

This comment is too similar to the comment in the class "FuzzyTokenizer". These are two different classes for different purposes, so their comments should be different.

Modify the comments accordingly.